example/image-classification/common/data.py

```diff
@@ -120,6 +136,7 @@ def get_rec_iter(args, kv=None):
     else:
         (rank, nworker) = (0, 1)
     rgb_mean = [float(i) for i in args.rgb_mean.split(',')]
+    random_resized_crop = args.random_resized_crop is not None
```

Comment:
Wouldn't args.random_resized_crop never be None? action='store_true' means if not passed it is set by default to false

Reply:
Right. I'll fix that.

Reply:
Also, for uniformity with random-crop and random-mirror we might want to random-resized-crop be of type int